csharp/ql/integration-tests/all-platforms/msbuild/test.py

```diff
@@ -0,0 +1,4 @@
+from create_database_utils import *
+
+# force CodeQL to use MSBuild by setting `LGTM_INDEX_MSBUILD_TARGET`
```

Comment:
We will soon get rid of the `LGTM_` env variables, so it would be better not to rely on them. It would be better to create a .NET Classic project, as the auto builder will then use `msbuild` instead of `dotnet build`.

Reply:
I tried that first, but I couldn't get it to work because `dotnet` doesn't support .NET classic. So then we need `mono` instead on macOS and that doesn't provide `msbuild` on arm64.

Reply:
OK, let's do this for now then.